client/servers.cpp

```diff
@@ -873,27 +667,14 @@ void server_scan_finish(struct server_scan *scan)
     fc_thread_wait(&scan->meta.thr);
     fc_destroy_mutex(&scan->meta.mutex);
 
-    /* This mainly duplicates code from below "else" block.
-     * That's intentional, since they will be completely different in future
-     * versions. We are better prepared for that by having them separately
-     * already. */
-    if (scan->sock >= 0) {
-      fc_closesocket(scan->sock);
-      scan->sock = -1;
-    }
```

Comment:
Missing `fcUdpScan::i()->drop();`?

Reply:
Its there in server scan finish line 677 ( no idea how to link code :D )
`  } else {
    fcUdpScan::i()->drop();`